ietf-microwave-types.yang

```diff
@@ -36,9 +36,9 @@ module ietf-microwave-types {
   // replace the revision date with the module publication date
   // the format is (year-month-day)
 
-  revision 2024-03-03 {
+  revision 2024-09-11 {
     description
-      "pretty printing";
+      "Draft - new grouping Issue #35";
```

Comment:
According to RFC8407 (and RFC8407-bis), we need only one revision statement describing the YANG model revision in RFC8561-bis and not multiple revisions for each draft version